### DISCUSSION TITLE
gitlab.owner required even when docs says its not

### DISCUSSION THREAD

**Discussion by javydreamercsw** [jreleaser/jreleaser#217] [Q&A] [answered]
Not defining gitlab.owner leads to this error even when the docs say it'll be obtained from the repo:

```
== JReleaser ==
gitlab.owner must not be blank
gitlab.owner must not be blank
```

**Comment by aalmiray** [comment, marked as answer]
That is indeed the case when the host == "gitlab.com" as witnessed by https://github.com/jreleaser/jreleaser/blob/73b8dc5b3ecd0617bc3401cce28948e99f4690bd/sdks/git-sdk/src/main/java/org/jreleaser/sdk/git/GitSdk.java#L75-L86

However I think that a custom GitLab host will make JReleaser skip the default configuration applied by

https://github.com/jreleaser/jreleaser/blob/73b8dc5b3ecd0617bc3401cce28948e99f4690bd/core/jreleaser-engine/src/main/java/org/jreleaser/engine/context/ModelConfigurer.java#L63-L73

This is indeed a bug as there's some defaults that can be safely applied.
